EventHandler.java

```diff
@@ -0,0 +1,170 @@
+package SearchDemo;
+
+import com.aliyun.fc.runtime.*;
+import com.aliyun.opensearch.DocumentClient;
+import com.aliyun.opensearch.OpenSearchClient;
+import com.aliyun.opensearch.sdk.dependencies.com.google.common.collect.Maps;
+import com.aliyun.opensearch.sdk.generated.OpenSearch;
+import com.aliyun.opensearch.sdk.generated.commons.OpenSearchClientException;
+import com.aliyun.opensearch.sdk.generated.commons.OpenSearchException;
+import com.aliyun.opensearch.sdk.generated.commons.OpenSearchResult;
+import com.aliyun.oss.OSSClient;
+import com.aliyun.oss.model.OSSObject;
+import net.sf.json.JSONArray;
+import net.sf.json.JSONObject;
+
+import java.io.*;
+import java.util.*;
+
+public class EventHandler implements StreamRequestHandler {
+
+    private static String ossEndpoint = "YourOSSEndpoint";
+    private static String openSearchAppName = "YourOpenSearchAppName";
+    private static String openSearchHost = "YourOpenSearchHost";
+    private static String openSearchTableName = "YourOpenSearchTableName";
+    private static String accessKeyId = "YourAccessKeyId";
+    private static String accessKeySecret = "YourAccessSecretId";
+    private static String docUrlFormat = "http://%s.%s/%s";
+
+    private static String[] addEventArray = {"ObjectCreated:PutObject", "ObjectCreated:PostObject"};
+    private static List<String> addEventList = Arrays.asList(addEventArray);
+    private static String[] updateEventArray = {"ObjectCreated:AppendObject"};
+    private static List<String> updateEventList = Arrays.asList(updateEventArray);
+    private static String[] deleteEventArray = {"ObjectRemoved:DeleteObject", "ObjectRemoved:DeleteObjects"};
+    private static List<String> deleteEventList = Arrays.asList(deleteEventArray);
```

Comment:
This can be simplified as deleteEventList = Arrays.asList("ObjectRemoved:DeleteObject", "ObjectRemoved:DeleteObjects");